src/main/resources/application.properties

```diff
@@ -45,7 +46,27 @@ get-HRP-Status=ANC/getHRPStatus
 #Get Beneficiary ABHA
 getHealthID=healthID/getBenhealthID
 
+# FHIR Config
+#fhir-url=<Enter your socket address here>/fhirapi-v1.0
+#server.port=8082
+
+# TM Config
+#tm-url=<Enter your socket address here>/tmapi-v1.0
+
+
+
+springdoc.api-docs.enabled=true
+springdoc.swagger-ui.enabled=true
 
+sendSMSUrl =
```

Comment:
These variables belong to common_ files in environment folder.
Please move them there.
We also don't want to enable swagger by default for everyone.

Please check how it's done in other repos.